enet.pyx

```diff
@@ -1000,6 +1042,27 @@ cdef class Host:
         def __set__(self, value):
             self._enet_host.totalReceivedPackets = value
 
+    property intercept:
+        def __get__(self):
+            return self._interceptCallback
```

Comment:
It's a C function, so unfortunately not. In this specific `property`, we need to operate with python functions because this is what user will be assigning/accessing from a `.py` script.